src/java.base/macosx/classes/sun/nio/fs/BsdNativeDispatcher.java

```diff
@@ -62,6 +64,28 @@ static byte[] getmntonname(UnixPath path) throws UnixException {
     }
     static native byte[] getmntonname0(long pathAddress) throws UnixException;
 
+    /**
+     * setattrlist(const char* path, struct attrlist* attrList, void* attrBuf,
+     *             size_t attrBufSize, unsigned long options)
+     */
+    static void setattrlist(UnixPath path, int commonattr, long modTime,
+                            long accTime, long createTime, long options)
+        throws UnixException {
```

Comment:
minor nit but the "{" at the end of line looks a bit odd here.

Reply:
Fixed in 10deb03d62b15eb0c5130db3e251838787a82e00.